### ISSUE TITLE
Client for pc

### ISSUE THREAD

**Issue by grevius5** [rivafarabi/deckboard#555]
When I stream I use another pc as a chat desktop (i have only one monitor on my pc). It would be great to have a client version of the app available from other browsers, in my example from the browser from the chat pc. I think it can be possible by converting the app template to a client version to show in the browser.

**Comment by chrisbadley** [comment]
Can you just use the "start" button at the top of the windows version now to turn it into a client?

**Comment by grevius5** [comment]
No, I have the app in the streaming pc but I want to use the app functionality from another pc.

**Comment by XsFrozdY** [comment]
I'm just throwing an idea here on how it could be implemented, we could have the "edit mode" and the "control mode" switch on the host.

So "edit mode" is what the host already is, you can edit your buttons.
"control mode" would be the "client control".

That logically seems like the best way of implementing this.

**Comment by chrisbadley** [comment]
> I'm just throwing an idea here on how it could be implemented, we could have the "edit mode" and the "control mode" switch on the host.
> 
> So "edit mode" is what the host already is, you can edit your buttons.
> "control mode" would be the "client control".
> 
> That logically seems like the best way of implementing this.

That already exists, his problem is that he wants a client only on a separate PC, presumably Windows OS, which doesn't exist and would require a new application making
